src/TLuaInterpreter.cpp

```diff
@@ -3203,92 +3203,212 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
     return 0;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
```

Comment:
`updateEvent` doesn't sound like a call to action, whereas this is exactly what this parameter is. How about `resizeMudlet` ?